manifests/params.pp

```diff
@@ -50,6 +50,8 @@
           'oddjob-mkhomedir',
         ]
         $extra_packages_ensure = 'present'
+        $absent_packages = []
+        $absent_packages_ensure = 'absent'
```

Comment:
Moving these two variables above the "if versioncmp" conditional would probably solve the current travis ci error on RedHat 5.11.